client/controller/client/http.go

```diff
@@ -111,7 +111,7 @@ func checkForErrors(res *http.Response, body string) error {
 	bodyMap := make(map[string]interface{})
 
 	if err := json.Unmarshal([]byte(body), &bodyMap); err != nil {
-		return err
+		return errors.New("\n" + body + "\n" + res.Status + "\n")
```

Comment:
You're right, that would be consistent with the python CLI. I'll see about putting the `response.Status` line first in all these cases.